doc/man/maestro-config.1

```diff
@@ -43,8 +43,6 @@ Since Maestro is in the prototypal phase, bug reports are welcome and can be sub
 the GitHub Issues page
 .UE .
 
-.SH EXAMPLE
-
```

Comment:
Do you think we could just have a quick example written here and showing what the prompt would look like?

Reply:
See latest commit.